src/components/StatsPanel/StatsPanel.styles.ts

```diff
@@ -0,0 +1,48 @@
+import { styled } from '@mui/material/styles';
+
+export const CardContainer = styled('div')(({ theme }) => ({
+  backgroundColor: theme.palette.background.paper,
+  borderRadius: '20px',
+  padding: theme.spacing(4),
+  display: 'flex',
+  flexDirection: 'column',
+  gap: theme.spacing(3),
+  width: '260px',
+  boxShadow: '0px 4px 10px rgba(0,0,0,0.1)',
+}));
+
+export const StatItem = styled('div')({
+  display: 'flex',
+  alignItems: 'center',
+  gap: '14px',
+});
+
+export const IconWrapper = styled('div')(({ theme }) => ({
+  width: '48px',
+  height: '48px',
+  borderRadius: '50%',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  backgroundColor: theme.palette.action.hover,
+  opacity: 0.9,
+}));
+
+export const StatIcon = styled('div')<{ color: string }>(({ color }) => ({
+  fontSize: '28px',
```

Comment:
fontSize is hard-coded to '28px' and other text sizes to '22px' / '14px'. Prefer theme.typography.fontSize or theme.typography variants (e.g., theme.typography.h6, body1) to keep consistency and accessibility.

**Sugerencia:**
```typescript
Use typography tokens: fontSize: theme.typography.h6.fontSize or apply variant props on MUI Typography components instead of hardcoding font sizes.
```